app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable, :confirmable
+
```

Comment:
- [ ] Good job adding `:confirmable` to your User model. However, as required for the for the project, you should ensure that User cannot be able to login before his `email` has been confirmed, because it is not the same in your case. Right now, after a user has singed up, he is immediately logged in even if without being confirmed. 

You can see an example on this picture after signing up, your application requests for confirmation instead the User has already logged in. Make sure you fix that in order your application to be approved.
![Screenshot from 2023-07-25 22-31-23](https://github.com/mnaibei/blog_app/assets/83042742/9888fa69-1be0-46ca-93ca-95cc1362db53)